packages/snaps-rpc-methods/src/permitted/createInterface.test.tsx

```diff
@@ -132,7 +132,7 @@ describe('snap_createInterface', () => {
       error: {
         code: -32602,
         message:
-          'Invalid params: At path: ui -- Expected the value to satisfy a union of `union | union`, but received: "foo".',
+          'Invalid params: At path: ui -- Expected the value to satisfy a union of `typedUnion | union`, but received: "foo".',
```

Comment:
You could name the struct `union` so this doesn't change. `typedUnion` doesn't make much sense to other people. Not that this entire message makes sense otherwise, but 🤷

Reply:
Yeah, I was considering that. The nice thing about using a separate type is that it becomes easier for us to identify it in the list of errors. But it may not be worth the extra confusion.

Reply:
done!